examples/experimental/CrypTen/CrypTen - Traing Encrypted Neural Network - Plans.ipynb

```diff
@@ -6,8 +6,7 @@
    "source": [
```

Comment:
<p>Any idea why time goes up here? is it the same env as before?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3951/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
There was added some more logic into the plans building process and the CrypTen commit which we use might not be the same one which when we did our benchmarking.
When I did the benchmarking I used the latest master and when I merged the changes I used the latest commit from master to pin it to that "version". It might not be the same.